source/trace-proto/src/lib.rs

```diff
@@ -2,11 +2,14 @@
 
 use core::{fmt, num::NonZeroU64};
 use tracing_serde_structured::{
-    SerializeId, SerializeMetadata, SerializeRecordFields, SerializeSpanFields,
+    SerializeId, SerializeLevel, SerializeMetadata, SerializeRecordFields, SerializeSpanFields,
 };
 
 #[derive(serde::Serialize, serde::Deserialize)]
 pub enum TraceEvent<'a> {
+    /// Sent by the target periodically when not actively tracing, to indicate
```

Comment:
We might want to mark this non-exhaustive and add things to the end instead, to avoid breaking wire format, once we care about that.

Reply:
yeah, i figured we didn't currently care about stability for the crate API _or_ for the wire format, but we can do that in the future.